lib/Onyx.js

```diff
@@ -150,6 +150,43 @@ function addAllSafeEvictionKeysToRecentlyAccessedList() {
         });
 }
 
+/**
+ * When a collection of keys change, search for any callbacks matching the collection key and trigger those callbacks
+ *
+ * @param {String} collectionKey
+ * @param {Object} collection
+ */
+function keysChanged(collectionKey, collection) {
+    // Find all subscribers that were added with connect() and trigger the callback or setState() with the new data
+    _.each(callbackToStateMapping, (subscriber) => {
+        if (subscriber
+            && isKeyMatch(subscriber.key, collectionKey)
+            && isCollectionKey(subscriber.key)
+        ) {
+            if (_.isFunction(subscriber.callback)) {
+                _.each(collection, (data) => {
+                    subscriber.callback(data, collectionKey);
```

Comment:
Probably the key in `subscriber.callback()` here should be the individual key not the collection key? That is what happens in `keyChanged()` here and we should consider keeping this behavior consistent.

https://github.com/Expensify/react-native-onyx/blob/4e810bfd30cbb02aaf663ba69456fd233346a774/lib/Onyx.js#L170-L172

Having said that, I don't recall whether anything is actually _using_ the `key` parameter in the callback (but there's basically no other way to tell which record in the collection you are seeing pop up in the callback so it makes sense for it to be there).

Reply:
Yup, you're correct. Shall update.